.github/workflows/web_tests.yml

```
@@ -6,20 +6,18 @@ permissions:
 jobs:
   web_tests:
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
```

FYI: We are not running a matrix here so failing fast has no effect.